### PR TITLE
Catalyst Support

### PR DESCRIPTION
**Pull request by FaizanDurrani**
It *works* but probably could use code cleanup

### CODE REVIEW

File: NineAnimator/Controllers/Settings Scene/TrackingServiceTableViewController.swift [hunk 1]
```diff
@@ -467,17 +467,6 @@ extension TrackingServiceTableViewController {
             // references
             _ = session.start()
             authenticationSessionReference = session
-        } else {
```
Comment (FaizanDurrani) [inline NineAnimator/Controllers/Settings Scene/TrackingServiceTableViewController.swift]: You can probably re-add this, it was crashing on catalyst so I just removed it all then didn't bother adding it back when I added the #available check

File: NineAnimator/NineAnimator.entitlements [hunk 2]
```diff
@@ -2,15 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>com.apple.developer.associated-domains</key>
```
Comment (FaizanDurrani) [inline NineAnimator/NineAnimator.entitlements]: Had to remove associated domains because free dev account doesn't support it

**Comment by FaizanDurrani** [comment]
I haven't removed the Cartfile, but it's probably a good idea to remove it so there's nothing left of Carthage.

The project also uses modified OpenCastSwift which is here https://github.com/FaizanDurrani/OpenSwiftCast

**Comment by Catta1997** [comment]
@FaizanDurrani One feature that I suggest is to implement the Touch Bar to navigate in Watch Next,Fetatured ...

**Comment by FaizanDurrani** [comment]
> @FaizanDurrani One feature that I suggest is to implement the Touch Bar to navigate in Watch Next,Fetatured ...

I am not entirely sure if that’s possible with catalyst. You’d have to rewrite the whole program with AppKit to get Mac specific stuff.

**Comment by SuperMarcus** [comment]
Closing in favor of the [spm branch](https://github.com/SuperMarcus/NineAnimator/tree/spm).

**Pull request closed without merging**